bastionzero/service/agents/agenttype/agenttype.go

```diff
@@ -0,0 +1,13 @@
+package agenttype
+
+//go:generate go run github.com/lindell/string-enumer -t AgentStatus -o ./generated.go .
```

Comment:
I think this was a typo. Change to `-t AgentType` and re-run `go generate ./...`